int32 car_state #小车状态机
bool is_rotation_finish #旋转完成标志位
int32 index #第几个点
